docs/getting_started/install_on_ibexa_cloud.md

```diff
@@ -83,3 +88,13 @@ git push -u <platform.sh-remote> master
 !!! note
 
     `master` is the Platform.sh name for the production branch.
+
+!!! caution
+
+    If you want to use the [Platform.sh CLI](https://docs.platform.sh/development/cli.html), you have to set it up for Ibexa Cloud.
+    Consider using the following alias `ibexa_cloud` with the right set up instead of `platform`:
```

Comment:
```suggestion
    If you want to use the [Platform.sh CLI](https://docs.platform.sh/development/cli.html), you have to set it up for Ibexa Cloud.
    Consider using the following alias `ibexa_cloud` with a dedicated setup instead of `platform`:
```

Reply:
So I suggest going with this.